pkg/operatorstatus/operatorstatus.go

```diff
@@ -61,23 +62,35 @@ type ClusterOperatorWrapper interface {
 	Create(state *configv1.ClusterOperator) (err error)
 }
 
+func (o *ClusterOperatorHandler) setOperatorStatusWithoutInterrogatingConfig(progressing configv1.ConditionStatus, cfg *v1.Config, reason string) {
```

Comment:
warrants a comment to the effect of "this always sets available=true/degraded=false"

Reply:
comment added